### PR TITLE
release 2.0.2

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Review by llowder — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,35 @@
...
+
+**Merged pull requests:**
+
+- Upgrade to GitHub-native Dependabot [\#126](https://github.com/voxpupuli/rspec-puppet-facts/pull/126) ([dependabot-preview[bot]](https://github.com/apps/dependabot-preview))
+- Version bump and CHANGELOG for v2.0.1 [\#125](https://github.com/voxpupuli/rspec-puppet-facts/pull/125) ([DavidS](https://github.com/DavidS))
+- Implement fact memoization [\#122](https://github.com/voxpupuli/rspec-puppet-facts/pull/122) ([ekohl](https://github.com/ekohl))
+
+## [2.0.0](https://github.com/voxpupuli/rspec-puppet-facts/tree/2.0.0) (2020-08-05)
+
+*the git tag for 2.0.1 is missing, it was probably created by hand*
```
Comment (ekohl) [inline CHANGELOG.md:25]: Can't we track this back? Perhaps @mcanevet still has it locally.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,35 @@
...
+
+- Cut a new release [\#120](https://github.com/voxpupuli/rspec-puppet-facts/issues/120)
+- Memoizing facts [\#114](https://github.com/voxpupuli/rspec-puppet-facts/issues/114)
+- Commit 21442e7 looks to introduce hard to debug behavior [\#97](https://github.com/voxpupuli/rspec-puppet-facts/issues/97)
+
+**Merged pull requests:**
+
+- Upgrade to GitHub-native Dependabot [\#126](https://github.com/voxpupuli/rspec-puppet-facts/pull/126) ([dependabot-preview[bot]](https://github.com/apps/dependabot-preview))
+- Version bump and CHANGELOG for v2.0.1 [\#125](https://github.com/voxpupuli/rspec-puppet-facts/pull/125) ([DavidS](https://github.com/DavidS))
+- Implement fact memoization [\#122](https://github.com/voxpupuli/rspec-puppet-facts/pull/122) ([ekohl](https://github.com/ekohl))
```
Comment (ekohl) [inline CHANGELOG.md:21]: I'm pretty sure this was part of 2.0.1

File: CHANGELOG.md (hunk 1 above)
Comment (bastelfreak) [reply CHANGELOG.md]: the tag is missing, @mcanevet  doesn't have it available. I recreated it with
```
GIT_COMMITTER_DATE="2021-01-09 00:00" git tag -s 2.0.1 5d87fa0990ba9bf23ee127c1870cb41a1f2f3960
```

File: CHANGELOG.md (hunk 2 above)
Comment (bastelfreak) [reply CHANGELOG.md:29]: fixed by creating the 2.0.1 tag

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,34 @@
...
+- Move facterversion\_obj declaration out of the loop [\#131](https://github.com/voxpupuli/rspec-puppet-facts/pull/131) ([ekohl](https://github.com/ekohl))
+- Upgrade to GitHub-native Dependabot [\#126](https://github.com/voxpupuli/rspec-puppet-facts/pull/126) ([dependabot-preview[bot]](https://github.com/apps/dependabot-preview))
+
+## [2.0.1](https://github.com/voxpupuli/rspec-puppet-facts/tree/2.0.1) (2021-01-09)
+
+[Full Changelog](https://github.com/voxpupuli/rspec-puppet-facts/compare/2.0.0...2.0.1)
+
+**Closed issues:**
+
+- Cut a new release [\#120](https://github.com/voxpupuli/rspec-puppet-facts/issues/120)
```
Comment (ekohl) [inline CHANGELOG.md:22]: I skipped this

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,34 @@
...
+- Memoizing facts [\#114](https://github.com/voxpupuli/rspec-puppet-facts/issues/114)
+- Commit 21442e7 looks to introduce hard to debug behavior [\#97](https://github.com/voxpupuli/rspec-puppet-facts/issues/97)
+
+**Merged pull requests:**
+
+- Version bump and CHANGELOG for v2.0.1 [\#125](https://github.com/voxpupuli/rspec-puppet-facts/pull/125) ([DavidS](https://github.com/DavidS))
```
Comment (ekohl) [inline CHANGELOG.md:28]: Skipped this too

**Review by ekohl — COMMENTED** [review]
Please regen. :+1: after that.

**Review by ekohl — APPROVED** [review]

**Pull request merged**